protocol/messenger_sync_settings.go

```diff
@@ -179,10 +181,35 @@ func (m *Messenger) startSettingsChangesLoop() {
 				case settings.Bio.GetReactName():
 					m.selfContact.Bio = s.Value.(string)
 					m.publishSelfContactSubscriptions(&SelfContactChangeEvent{BioChanged: true})
+				case settings.LogLevel.GetReactName():
+					level := s.Value.(string)
+					m.updateLogConfig(level, logger)
 				}
 			case <-m.quit:
 				return
 			}
 		}
 	}()
 }
+
+func (m *Messenger) updateLogConfig(level string, logger *zap.Logger) {
+	m.setLogLevel(level, logger)
+	m.setLogEnabled(level != "", logger)
+}
+
+func (m *Messenger) setLogLevel(level string, logger *zap.Logger) {
+	if level == "" {
+		return
+	}
+	err := nodecfg.SetLogLevel(m.database, level)
+	if err != nil {
+		logger.Error("nodecfg.SetLogLevel", zap.Error(err))
+	}
+}
```

Comment:
There is such a method already:
https://github.com/status-im/status-go/blob/50933aa3286e54bfd407f109ce21eabf1417bae7/protocol/messenger_settings.go#L33-L39

Reply:
@qfrank but.. why did log end up being disabled?